src/olympia/lib/settings_base.py

```diff
@@ -1816,6 +1816,8 @@ def get_raven_release():
 
 MIGRATED_LWT_UPDATES_ENABLED = True
 
+ADDON_UPLOAD_RATE_LIMITS_BYPASS_EMAILS = ('addons+release@mozilla.org',)
```

Comment:
Just confirming this email is correct because I've not seen it before.  (Maybe it should be a permission group we add users to, rather than a hardcoded email address?)

Reply:
Was confirmed with @Callek on slack. I thought about doing a group+permission but I kinda wanted to restrict it more, I'd like to make sure the list is always under control. I don't see any valid use cases other than langpacks submission for new Firefox releases atm.